src/serialization/src/main/kotlin/org/vaccineimpact/api/serialization/DataTableDeserializer.kt

```diff
@@ -85,11 +85,18 @@ open class DataTableDeserializer<out T>(
         }
         catch (e: Exception)
         {
+            if (e is UnsupportedOperationException)
+            {
+                throw e
+            }
```

Comment:
Could you write this as a separate catch block that comes before the general case catch, rather than as an if statement?

Reply:
It can't come before this catch block, otherwise it'll still be caught by this one!

Reply:
Good point :)